utils/exporters/blender/addons/io_three/exporter/api/object.py

```diff
@@ -374,7 +378,7 @@ def matrix(obj, options):
         parent_inverted = obj.parent.matrix_world.inverted(mathutils.Matrix())
         return parent_inverted * obj.matrix_world
     else:
-        return AXIS_CONVERSION * obj.matrix_world
```

Comment:
Correct, we're not using AXIS_CONVERSION anymore. It was incorrectly converting the transform matrix on the skinned meshes--it worked ok on static meshes--so I removed it and the manually flipped the YZ axis as we did with an older version of the exporter. 

However, I kept it in there because we may want to go back to that method. Do we want to keep using an axis conversion matrix to apply a transform at run-time? Is there an advantage to doing it this way instead of grabbing the individual vert/bone transforms? What was the thought process behind that method?

The normals were broken because I forgot to do the same fix that I did to the verts.